plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/packagematerial/JsonMessageHandler1_0.java

```diff
@@ -159,7 +159,10 @@ public String requestMessageForLatestRevision(PackageConfiguration packageConfig
 
     @Override
     public PackageRevision responseMessageForLatestRevision(String responseBody) {
-        return toPackageRevision(responseBody);
+        PackageRevision packageRevision = toPackageRevision(responseBody);
+        if( packageRevision == null) {
+            throw new RuntimeException("Empty response body");
+        } else return packageRevision;
```

Comment:
Weirdly enough, the IntelliJ reformat did not fix this, for me. We should have a format specification file (for IntelliJ) which everyone can import, right? That would help in cases like this.